lib/sync-DataSetModel.js

```diff
@@ -371,7 +390,8 @@ var self = {
 
       var hashes = [];
       var recOut = {};
-      for (var i in records) {
+      /*for (var i in records) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)